app/models/katello/ping.rb

```diff
@@ -40,7 +38,10 @@ def ping_pulp_without_auth(service_result, capsule_id)
       def ping_candlepin_without_auth(service_result)
         url = SETTINGS[:katello][:candlepin][:url]
         exception_watch(service_result) do
-          backend_status(url, :candlepin)
+          result = backend_status(url, :candlepin)
+          if result[:mode] == 'SUSPEND'
+            fail _("Candlepin was in suspend mode")
+          end
```

Comment:
Are there any other 'modes' we should worry about?

Reply:
Looks like NORMAL, SUSPEND are all we have

Reply:
I guess we could check mode != NORMAL?

Reply:
I think i might prefer that, although you might ask the candlepin team which they see as more future proof

Reply:
Sounds like that is the better way to go - updated